src/app/examples/list-example/list-example.component.ts

```diff
@@ -7,9 +7,25 @@ import { Component, OnInit } from '@angular/core';
 })
 export class ListExampleComponent implements OnInit {
 
+  // myItems = [
+  //   {title: 'Netflix', amount: -79.95, detail: 'Peter'},
+  //   {title: 'McDonnalds Århus', amount: -20, detail: 'Jack'},
+  //   {title: 'Super Brugsen', amount: -180.65, detail: 'Morten'}
+  // ];
+
+  myItems = [
```

Comment:
Omdøb gerne til `items` så det er konsistent med vores andre eksempler:
```suggestion
   items = [
```